doc/classes/TextEdit.xml

```diff
@@ -262,6 +262,33 @@
 				Returns whether the line at the specified index is hidden or not.
 			</description>
 		</method>
+		<method name="is_line_set_as_bookmark" qualifiers="const">
+			<return type="bool">
+			</return>
+			<argument index="0" name="line" type="int">
+			</argument>
+			<description>
+			Returns [code]true[/code] when the specified [code]line[/code] is bookmarked.
```

Comment:
These should be indented one level more. Running `doctool` should fix it for you.